### DISCUSSION TITLE
GitHub Education does not recognize my email even though it is already accredited

### DISCUSSION THREAD

**Discussion by isavs** [community/community#55055] [GitHub Education]
### Hi everyone,

Hello, this is the second discussion I open about this (the first was the #54200 ), but since I didn't get a response, here goes:

I'm using my university email, but it's not working.

The follow warning keeps appearing as I try to continue:

![image](https://github.com/community/community/assets/33363682/f3884d80-dcb7-45e8-8c73-e4ac397830fa)

**Comment by Ntshangase** [comment]
Hello @isavs 

I would suggest that you apply to get your school to be a partner school using the link provided in the rejection reason.

**Reply by isavs** [reply]
Forgot to mention, I'm actually applying for academic reverification... My first application was with the same email as now

![image](https://github.com/community/community/assets/33363682/432b7664-dd00-48ba-b1d8-3a2253b4c27a)

**Reply by Ntshangase** [reply]
have you tried to apply using a different device?

**Reply by isavs** [reply]
yes, i tried it with different devices and different browsers :(

**Reply by Ntshangase** [reply]
ok, then maybe for some reason your school is no longer supported, I think you should go ahead and try get your school to be a partner.

**Reply by isavs** [reply]
ok then, i will contact them. ty

**Comment by isavs** [comment]
if I try to verify by other ways

![image](https://github.com/community/community/assets/33363682/88057051-cb06-47ed-8b6a-343a4f67c41a)
